<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAtcStationComponent</class>
 <widget class="QFrame" name="CAtcStationComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATC stations</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AtcStationComponent">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="sp_AtcSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tw_Atc">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tb_AtcStationsOnline">
       <attribute name="title">
        <string>ATC stations</string>
       </attribute>
       <layout class="QVBoxLayout" name="vl_AtcStationsOnline">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::views::CAtcStationView" name="tvp_AtcStationsOnline">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tb_AtcStationsOnlineTree">
       <attribute name="title">
        <string>ATC tree</string>
       </attribute>
       <layout class="QVBoxLayout" name="vl_AtcTree">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::views::CAtcStationTreeView" name="tvp_AtcStationsOnlineTree">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gb_Details">
      <property name="title">
       <string>Details (METAR, ATIS)</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="vl_Details">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QWidget" name="wi_ButtonsAndEntry" native="true">
         <layout class="QHBoxLayout" name="hl_ButtonsAndEntry">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="le_AtcStationsOnlineMetar">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string extracomment="METAR: Airport ICAO code">METAR: Airport ICAO code</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
            <property name="placeholderText">
             <string>e.g. KLAX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wi_Buttons" native="true">
            <layout class="QHBoxLayout" name="hl_Buttons">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="tb_AtcStationsLoadMetar">
               <property name="toolTip">
                <string>Load METAR</string>
               </property>
               <property name="text">
                <string>METAR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tb_AtcStationsAtisReload">
               <property name="toolTip">
                <string>Reload ATIS</string>
               </property>
               <property name="text">
                <string>ATIS</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tb_TextMessageOverlay">
               <property name="toolTip">
                <string>text message</string>
               </property>
               <property name="text">
                <string>tm</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tb_Audio">
               <property name="toolTip">
                <string>goto audio panel</string>
               </property>
               <property name="text">
                <string>audio</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="te_AtcStationsOnlineInfo">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CAtcStationView</class>
   <extends>QTableView</extends>
   <header>gui/views/atcstationview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::views::CAtcStationTreeView</class>
   <extends>QTreeView</extends>
   <header>gui/views/atcstationtreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tw_Atc</tabstop>
  <tabstop>tvp_AtcStationsOnline</tabstop>
  <tabstop>gb_Details</tabstop>
  <tabstop>le_AtcStationsOnlineMetar</tabstop>
  <tabstop>tb_AtcStationsLoadMetar</tabstop>
  <tabstop>tb_AtcStationsAtisReload</tabstop>
  <tabstop>tb_TextMessageOverlay</tabstop>
  <tabstop>tb_Audio</tabstop>
  <tabstop>te_AtcStationsOnlineInfo</tabstop>
  <tabstop>tvp_AtcStationsOnlineTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
